datasets/subjqa/README.md

```diff
@@ -0,0 +1,234 @@
+* ```nn_mod```: The modifier of the neighboring opinion (which appears in the review).
+* ```nn_asp```: The aspect of the neighboring opinion (which appears in the review).
+* ```query_mod```: The modifier of the query opinion (around which a question is manually written).
+* ```query_asp```: The aspect of the query opinion (around which a question is manually written).
+* ```item_id```: The id of the item/business discussed in the review.
+* ```review_id```: A unique id associated with the review.
+* ```q_review_id```: A unique id assigned to the question-review pair.
+* ```q_reviews_id```: A unique id assigned to all question-review pairs with a shared question.
+
+### Data Splits
+
+Each domain consists of three splits: train, dev, and test.
+
+## Dataset Creation
+
+### Curation Rationale
+
+[Needs More Information]
+
+### Source Data
+
+#### Initial Data Collection and Normalization
+
+[Needs More Information]
+
+#### Who are the source language producers?
+
+[Needs More Information]
+
+### Annotations
+
+#### Annotation process
+
+[Needs More Information]
+
+#### Who are the annotators?
+
+[Needs More Information]
+
+### Personal and Sensitive Information
+
+[Needs More Information]
+
+## Considerations for Using the Data
+
+### Social Impact of Dataset
+
+[Needs More Information]
+
+### Discussion of Biases
+
+[Needs More Information]
+
+### Other Known Limitations
+
+[Needs More Information]
+
+## Additional Information
+
+### Dataset Curators
+
+[Needs More Information]
+
+### Licensing Information
+
+The SubjQA dataset is provided "as-is", and its creators make no
+represenntation as to its accuracy. Furthermore, the creators
+have on obligation to maintain the dataset.
+
+The SubjQA dataset is constructed based on the following datasets and thus contains subsets of their data:
+* [Amazon Review Dataset](http://jmcauley.ucsd.edu/data/amazon/links.html) from UCSD
+    * Used for _books_, _movies_, _grocery_, and _electronics_ domains
+* [The TripAdvisor Dataset](http://times.cs.uiuc.edu/~wang296/Data/) from UIUC's Database and Information Systems Laboratory
+    * Used for the _TripAdvisor_ domain
+* [The Yelp Dataset](https://www.yelp.com/dataset)
+    * Used for the _restaurants_ domain
+
+Consequently, the data within each domain of the SubjQA dataset should be considered under the same license as the dataset it was built upon.
+
+### Citation Information
+
+If you are using the dataset, please cite the following in your work:
+```
+@inproceedings{bjerva20subjqa,
+    title = "SubjQA: A Dataset for Subjectivity and Review Comprehension",
+    author = "Bjerva, Johannes  and
+      Bhutani, Nikita  and
+      Golahn, Behzad  and
+      Tan, Wang-Chiew  and
+      Augenstein, Isabelle",
+    booktitle = "Proceedings of the 2020 Conference on Empirical Methods in Natural Language Processing",
+    month = November,
+    year = "2020",
+    publisher = "Association for Computational Linguistics",
+}
+```
```

Comment:
````suggestion
```

### Contributions

Thanks to [@lewtun](https://github.com/lewtun) for adding this dataset.

````